File: pinyougou-parent/pinyougou-content-interface/src/main/java/com/pinyougou/content/service/ContentService.java
package com.pinyougou.content.service;

import com.pinyougou.pojo.TbContent;
import entity.PageResult;

import java.util.List;

public interface ContentService {

    PageResult<TbContent> findByPage(TbContent content, int pageNum, int pageSize) throws Exception;

    TbContent findById(Long id) throws Exception;

    void save(TbContent content) throws Exception;

    void delete(Long[] ids) throws Exception;

    void updateStatus(Long[] ids, String status) throws Exception;

    List<TbContent> findByCategoryId(Long categoryId) throws Exception;
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/plugins/angularjs/service/contentService.js
app.service('contentService', function ($http) {

    this.findByPage = function (page, size, searchEntity) {
        return $http.post('../content/findByPage.do?page=' + page + '&size=' + size, searchEntity);
    };

    this.findById = function (id) {
        return $http.get('../content/findById.do?id=' + id);
    };

    this.save = function (entity) {
        return $http.post('../content/save.do', entity);
    };

    this.del = function (ids) {
        return $http.get('../content/del.do?ids=' + ids);
    };

    this.updateStatus = function (ids, status) {
        return $http.post('../content/updateStatus/' + status + '.do?ids=' + ids)
    };
});